tevm/docs/state/functions/getContractStorage.md

```
@@ -8,7 +8,11 @@
 
 > **getContractStorage**(`baseState`): (`address`, `key`) => `Promise`\<`Uint8Array`\<`ArrayBufferLike`\>\>
 
+<<<<<<< HEAD
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: atx; Actual: setext

(MD003, heading-style)